app/models/metadata_field/boolean.rb

```diff
@@ -0,0 +1,9 @@
+class MetadataField::Boolean < MetadataField
+  TYPE = 'true/false'.freeze
+  # TODO: Actually use below options array, it should be used to generate the select options in the frontend
+  # Once in place it can be used for ENUM types, just will be dynamically created options rather than static
+  OPTIONS = [
```

Comment:
This options array actually doesn't matter b/c we can just split 'true/false' on / and get the same information.  (Assigning true=1 etc. doesn't matter because we aren't doing boolean algebra -- just matching)  This way our enum type can just be sent up as EG 'Read/Unread/Reading' and check if it is an enum based on presence of /. (boolean is essentially an enum)